src/submodules/io/jld2_integration.jl

```diff
@@ -54,10 +54,9 @@ struct SerializableABM{S,A<:AbstractAgent,P,R<:AbstractRNG}
     maxid::Int64
 end
 
-struct SerializableGridSpace{D,P,W}
+struct SerializableGridSpace{D,P}
```

Comment:
Can we do another 'pass the jld2 model around via email' test to make sure this is working fine?

Reply:
All good.